meilisearch-error/src/lib.rs

```diff
@@ -38,8 +38,8 @@ impl fmt::Display for ErrorType {
 
         match self {
             InternalError => write!(f, "internal_error"),
```

Comment:
```suggestion
            InternalError => write!(f, "internal_error_error"),
```

Reply:
(maybe the displayed string is not here)

Reply:
`internal_error` should be enough I think

Reply:
I didn't think at all, I was like a robot